### PR TITLE
Added support for multiple overrun handlers and fixed the tests

### PR DESCRIPTION
**Pull request by HernanRivasAcosta**
@elbrujohalcon I would say you need to review this PR's SUITE extra carefully.

### CODE REVIEW

File: test/wpool_SUITE.erl [hunk 1]
```diff
@@ -104,12 +119,13 @@ too_much_overrun(_Config) ->
   TCPid ! {check, Worker, TaskId, 100}, % tiny runtime, to check
 
   ct:comment("Nothing happens..."),
-  receive
-  after 1000 -> ok
-  end,
+  _ = case get_messages(1000) of
```
Comment (elvisci) [inline test/wpool_SUITE.erl]: The code in the following (LINE, COL) locations has the same structure: (122, 5), (156, 5).

Comment (HernanRivasAcosta) [inline test/wpool_SUITE.erl]: This warning should not appear on SUITES

Comment (elbrujohalcon) [inline test/wpool_SUITE.erl]: @HernanRivasAcosta you can always configure elvis (through elvis.config) appropriately.

File: test/wpool_SUITE.erl [hunk 2]
```diff
@@ -81,21 +83,34 @@ too_much_overrun(_Config) ->
...
-  end,
+  _ = receive
+        {overrun1, Message1} ->
+          overrun = proplists:get_value(alert, Message1),
+          wpool_SUITE_too_much_overrun = proplists:get_value(pool, Message1),
+          Worker  = proplists:get_value(worker, Message1),
+          {cast, {timer, sleep, [5000]}} = proplists:get_value(task, Message1),
+          9999999999 = proplists:get_value(runtime, Message1)
+      after 100 ->
+        throw(no_overrun)
```
Comment (elbrujohalcon) [inline test/wpool_SUITE.erl]: It would be better to use `ct:fail` here.

File: test/wpool_SUITE.erl [hunk 3]
```diff
@@ -81,21 +83,34 @@ too_much_overrun(_Config) ->
...
+  ct:comment("Get overrun message..."),
+  _ = receive
+        {overrun2, Message2} ->
+          overrun = proplists:get_value(alert, Message2),
+          wpool_SUITE_too_much_overrun = proplists:get_value(pool, Message2),
+          Worker  = proplists:get_value(worker, Message2),
+          {cast, {timer, sleep, [5000]}} = proplists:get_value(task, Message2),
+          9999999999 = proplists:get_value(runtime, Message2)
+      after 100 ->
+        throw(no_overrun)
```
Comment (elbrujohalcon) [inline test/wpool_SUITE.erl]: It would be better to use `ct:fail` here.

File: test/wpool_SUITE.erl [hunk 4]
```diff
@@ -81,21 +83,34 @@ too_much_overrun(_Config) ->
...
+      end,
 
   ct:comment("No more overruns..."),
-  receive
-  after 1500 -> ok
-  end,
+  _ = case get_messages(100) of
+        []    -> ok;
+        Msgs1 -> throw({unexpected_messages, Msgs1})
```
Comment (elbrujohalcon) [inline test/wpool_SUITE.erl]: It would be better to use `ct:fail` here.

File: test/wpool_SUITE.erl [hunk 5]
```diff
@@ -104,12 +119,13 @@ too_much_overrun(_Config) ->
   TCPid ! {check, Worker, TaskId, 100}, % tiny runtime, to check
 
   ct:comment("Nothing happens..."),
-  receive
-  after 1000 -> ok
-  end,
+  _ = case get_messages(1000) of
+        []    -> ok;
+        Msgs2 -> throw({unexpected_messages, Msgs2})
```
Comment (elbrujohalcon) [inline test/wpool_SUITE.erl]: It would be better to use `ct:fail` here.

File: test/wpool_SUITE.erl [hunk 6]
```diff
@@ -118,56 +134,58 @@ overrun(_Config) ->
...
+        {overrun1, Message} ->
+          overrun = proplists:get_value(alert, Message),
+          wpool_SUITE_overrun_pool = proplists:get_value(pool, Message),
+          WPid = proplists:get_value(worker, Message),
+          true = is_pid(WPid),
+          {cast, {timer, sleep, [1500]}} = proplists:get_value(task, Message),
+          Runtime = proplists:get_value(runtime, Message),
+          true = Runtime >= 1000
+      after 1500 ->
+        throw(no_overrun)
```
Comment (elbrujohalcon) [inline test/wpool_SUITE.erl]: It would be better to use `ct:fail` here.

File: test/wpool_SUITE.erl [hunk 7]
```diff
@@ -118,56 +134,58 @@ overrun(_Config) ->
...
+      end,
+
+  _ = case get_messages(1000) of
+        []   -> ok;
+        Msgs -> throw({unexpected_messages, Msgs})
```
Comment (elbrujohalcon) [inline test/wpool_SUITE.erl]: It would be better to use `ct:fail` here.

**Pull request merged**